### PR TITLE
Allow psr/cache v3.0

### PR DESCRIPTION
**Pull request by avrphp**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "ext-sockets": "*",
         "opentracing/opentracing": "^1.0",
         "packaged/thrift": "^0.13",
-        "psr/cache": "^1.0 || ^2.0",
+        "psr/cache": "^1.0 || ^2.0" || ^3.0",
```
Comment (sergeyklay) [inline composer.json:30]:
```suggestion
        "psr/cache": "^1.0 || ^2.0 || ^3.0",
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
@avrphp Could you please fix mentioned typo, and also this branch has conflicts that must be resolved. Could you rebase?

**Comment by avrphp** [comment]
@sergeyklay done

**Review by sergeyklay — APPROVED** [review]
Looks good to me. Thank you for contributing!

**Comment by sergeyklay** [comment]
Just released new version: https://github.com/jonahgeorge/jaeger-client-php/releases/tag/v1.4.3

**Pull request merged**